common/actions/swap.js

```diff
@@ -1,10 +1,22 @@
 // @flow
-
-/*** Change Step ***/
-export type ChangeStepSwapAction = {
-  type: 'SWAP_STEP',
-  value: number
-};
+import type {
+  OriginKindSwapAction,
+  DestinationKindSwapAction,
+  OriginAmountSwapAction,
+  DestinationAmountSwapAction,
+  LoadBityRatesSucceededSwapAction,
+  DestinationAddressSwapAction,
+  OrderCreateSucceededSwapAction,
+  OrderCreateRequestedSwapAction,
+  OrderStatusSucceededSwapAction,
+  ChangeStepSwapAction,
+  Pairs,
+  RestartSwapAction,
+  LoadBityRatesRequestedSwapAction,
+  StopLoadBityRatesSwapAction,
+  BityOrderResponse,
+  BityOrderPostResponse
+} from './swapTypes';
```

Comment:
It was changed before you made the big redux/flow refactor, but I also like it a bit better if they're separate when there's like 15+ action creators just due to size.